Service/UpdateChecklistService.php

```diff
@@ -2,19 +2,39 @@
 
 namespace CommonGateway\HuwelijksplannerBundle\Service;
 
+use App\Entity\Entity as Schema;
+use App\Entity\ObjectEntity;
+use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
 /**
  * This service holds al the logic for checksing data from the marriage request and updating the associated checklist.
  */
 class UpdateChecklistService
 {
+    /**
+     * @var EntityManagerInterface
+     */
+    private EntityManagerInterface $entityManager;
+
+    /**
+     * @var SymfonyStyle
+     */
     private SymfonyStyle $io;
```

Comment:
Codacy: "$io is a very short name, variable names should be at least 3 characters long"

Maybe use $symfonyStyle ?